src/components/Dashboard/Dashboard.jsx

```diff
@@ -33,39 +29,43 @@ class Dashboard extends React.Component {
             console.log(err);
         });
     }
+
     endTurn() {
-        console.log("endTurn");
         axios.get('/api/endTurn')
-        .then(response => {
-            console.log(response);
-        })
         .catch(err => {
             console.log(err);
-        })
+        });
+    }
+
+    componentDidMount() {
+        // If log in succeeds, connect to the socket
+        const socket = connectSocket();
+        dispatch(setSocket(socket));
     }
+    
     render() {
         let currentUsername = 'no one';
         if (this.props.gameState.currentPlayer) {
             currentUsername = this.props.gameState.currentPlayer.username;
         }        
         return (
             <div className={style.roundInfo}>
-                <h2>Round: {this.props.gameState.roundNum}</h2>
+                {this.props.gameState.started && <h2>Round: <span id="round-number">{this.props.gameState.roundNum}</span></h2>}
                 <div className={style.currentPlayersBox}>
                     Players:
-                    <ul>
+                    <ul id="players">
                         {this.props.gameState.players.map((player, key) => {
                             return <li key={key}>{player.username}</li>
                         })}
                     </ul>
                 </div>
                 <h2>Current player: {this.props.gameState.currentPlayer && <span id="current-player">{currentUsername}</span>}</h2>
-                {!this.props.gameState.started && <button onClick={this.startGame}>Start game</button>}
-                <button onClick={this.endTurn}>Next turn</button>
+                {!this.props.gameState.started && <button id="start-game" onClick={this.startGame}>Start game</button>}
+                {this.props.gameState.started && <button id="end-turn" onClick={this.endTurn}>Next turn</button>}
```

Comment:
I would consider re-wording this as a ternary. I.e.:
{this.props.gameState.started ? `next turn button` : `start game button`}
Or similar